ChurchCRM/CRM
# Bugfix/4794 duplicate family count query

#### What's this PR do?
Removes a duplicated and non-functional query from the `query_qry` table.

#### Screenshots (if appropriate)
Screen shot of **corrected** query listing - notice single "Family Member Count"
![Screen Shot 2019-04-09 at 9 56 25 am](https://user-images.githubusercontent.com/6720584/55764189-e50a2480-5aad-11e9-9fff-c7900a85a74b.png)

Screen shot of working query.
![Screen Shot 2019-04-09 at 9 49 11 am](https://user-images.githubusercontent.com/6720584/55764139-a07e8900-5aad-11e9-9df5-db2de180de86.png)

#### What Issues does it Close?

Closes #4794 

#### What are the relevant tickets?

Only #4794 

#### Any background context you want to provide?

User reported the first query failed and after testing it was clear this query would not work. A newer "Family Member Count" query was available but having a functional and non-functional query on the same page is confusing.

#### Where should the reviewer start?

Open the http://192.168.33.10/QueryList.php page

#### How should this be manually tested?

0. Add at least one family to the database and **do not run the SQL changes**.
1. Run the first "Family Member Count" query (ie, `http://192.168.33.10/QueryView.php?QueryID=1`) - it will return no results.
2. Run the second "Family Member Count" query (ie, `http://192.168.33.10/QueryView.php?QueryID=3`) - it will return the expected number of members for each family as per the description.
3. Now run the SQL changes in the `3.3.3.sql` file.
4. http://192.168.33.10/QueryList.php will now only show a single functional "Family Member Count" query as per screen shots.

#### How should the automated tests treat this?

No special handling required. This PR removes broken features and the existing working features are handled correctly.

#### Questions:
- Is there a related website / article to substantiate / explain this change?

- Does the development wiki need an update?

- Does the user documentation wiki need an update?

- Does this add new dependencies?
  -  [x] No - except for a database change on upgrade.

- Does this need to add new data to the demo database
  -  [x] No - but the old query (`qry_ID=1`) needs to be removed as per the `3.3.3.sql` file.
